Bottom: ea4d7deb6a7d9a5a9149a2675e3e3acb9bc1658d
Top:    d955eaf6b5c14e292de7c2e88d766ffe3dcf1c4c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 21:07:16 +0000

Refresh of well-on-the-way-to-completely

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index f9aed15..bfd0875 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -465,6 +465,7 @@ object3d_from_board_outline (void)
 #endif
           edge_info_set_stitch (UNDIR_DATA (edges[2 * npoints + i]));
         }
+    }
 
     if (1) {
       /* Cylinder centers on 45x45mm, stitch vertex is at 40x45mm. Radius is thus 5mm */
